app/pipeline/chat/exercise_chat_agent_pipeline.py

```
@@ -39,8 +39,11 @@
 from ...llm import CapabilityRequestHandler, RequirementList
 from ...llm import CompletionArguments
 from ...llm.langchain import IrisLangchainChatModel
+from ...retrieval.faq_retrieval import FaqRetrieval
+from ...retrieval.faq_retrieval_utils import should_allow_faq_tool, format_faqs
 from ...retrieval.lecture_retrieval import LectureRetrieval
 from ...vector_database.database import VectorDatabase
+from ...vector_database.faq_schema import FaqSchema
```

Ruff (0.8.2)

`...vector_database.faq_schema.FaqSchema` imported but unused

Remove unused import: `...vector_database.faq_schema.FaqSchema`

(F401)